docs/guide/how-does-it-work.md

```diff
@@ -1 +1,6 @@
 # How does it work?
+
+## Caveats
+
+- Rapier set object to sleep...
+- HMR sometimes work unexpected prefer reload page (unles reactive property is set)
```

Comment:
We have to figure out how to optimize this behavior 🤔

Reply:
Maybe not. I believe there's a PR (if not in events) to handle sleep and wake events.

But if not, this is the default behaviour of rapier. I think we should try to let the user decide.

In any case is not of the scope of this PR :)

Reply:
Yeah, this was just a comment. It's definitely not in scope of this PR